business/layer.go

```diff
@@ -51,8 +51,8 @@ func NewWithBackends(k8s kubernetes.IstioClientInterface, prom prometheus.Client
 	temporaryLayer.Health = HealthService{prom: prom, k8s: k8s}
 	temporaryLayer.Svc = SvcService{prom: prom, k8s: k8s, businessLayer: temporaryLayer}
 	temporaryLayer.IstioConfig = IstioConfigService{k8s: k8s}
-	temporaryLayer.Workload = WorkloadService{k8s: k8s}
 	temporaryLayer.Validations = IstioValidationsService{k8s: k8s, ws: temporaryLayer.Workload}
+	temporaryLayer.Workload = WorkloadService{k8s: k8s, prom: prom, businessLayer: temporaryLayer}
```

Comment:
Here comes the main problem of this PR. `ws` field from `IstioValidationsService` is not a pointer, so it is ws got a copy of `temporaryLayer` that was an empty object. So anytime we wanted to access to any page that consumed a IstioValidations an error raised.
This is fixed in: https://github.com/kiali/kiali/pull/684/files#diff-59657b2347e4d9f79961f3c52ce83db9R54